### ISSUE TITLE
Following the steps at README.md doesn't produces the documented result. Something is wrong, perhaps me or perhaps the program.

### ISSUE THREAD

**Issue by vituchon** [i18next/i18next-scanner#235]
The current documentation states this:

![image](https://user-images.githubusercontent.com/1927644/186000011-775d683f-7a9d-4b73-8343-ea5b6e7b454a.png)

I create a file called pepe.js with this content

![image](https://user-images.githubusercontent.com/1927644/186000104-2a18a45a-4899-486d-b7ad-b9ac7e079c74.png)

Then i create the config file taken from [here](https://github.com/i18next/i18next-scanner#examples).

You may note that I had to remove `const chalk = require('chalk');` because I don't know how to install I'm not the only, in matter of fact this is already [reported here](https://github.com/i18next/i18next-scanner/issues/152). I guess you need to clarify what other dependencie to install, BTW: `using sudo npm install` -g chalk DOESNT WORKS.

The i ran it using: `$i18next-scanner pepe.js` and the result was

![image](https://user-images.githubusercontent.com/1927644/186000488-d9178133-362c-4494-963f-0f5e6a43a584.png).

The observed result is not like the observed one in the README.md

To summarize I try my best, but after 4 hours of having no result, I'm afraid I will start to work with another tool for better results.

Thanks anyway for the code, and I guess it need some mantaince. The documentation is also very slim, I don't get most of the options  and what are their purpose.

**Comment by vituchon** [comment]
I really wonder if this project is actually manned by someone.....
